packages/stark-ui/src/modules/app-sidebar/services/app-sidebar.service.ts

```diff
@@ -0,0 +1,80 @@
+import { Injectable } from "@angular/core";
+import { Observable, Subject } from "rxjs";
+
+/**
+ * StarkOpenSidebarEvent interface
+ */
+export interface StarkOpenSidebarEvent {
+	mode?: "regular" | "menu";
+	sidebar: "left" | "right";
+}
+
+/**
+ * StarkAppSidebarService service
+ */
+@Injectable()
+export class StarkAppSidebarService {
```

Comment:
In Stark-Core we have an interface beside the the service implementation everytime. 
Could you please do the same here ?